src/components/search/SearchItemStats.tsx

```diff
@@ -0,0 +1,50 @@
+import RatingCapsule from '@components/search/RatingCapsule'
+import { DisplayableBusiness, Location } from '@typings/types'
+import React from 'react'
+import { Image, StyleSheet, Text, View } from 'react-native'
+
+// CODE REQUIRED -> Fetch user location and business location and calculate distance between
+function convertLocationToDistance(businessLocation: Location): number {
+  return 1.2
+}
+
+const SearchItemStats = ({business}: {business: DisplayableBusiness}) => (
+  <View style={styles.searchItemStats}>
+    <Text style={styles.businessTitleText}>{business.name}</Text>
+    <View style = {{flexDirection: 'row'}}>
+      <Image source={require('../../../assets/icon_location.png')} style={{width: 20, height: 20}} resizeMode="contain"/>
+      <Text style={styles.subtitleText}>{'0.0' + ' miles'}</Text>
```

Comment:
Use result of distance function.

Reply:
I've removed the distance function as this is something we'll need to review at a later date.